Jenkinsfile

```diff
@@ -132,6 +130,43 @@ pipeline {
          junit 'results/results.xml'
        }
      }
-   }
+    }
+
+    // when env.BRANCH_NAME == 'master', we will update the mesosphere:dcos-ui/latest branch
+    // when env.BRANCH_NAME =~ /(tag-name-regex)/, we will open a release PR against dcos/dcos
+    stage('Upload Build') {
+      when {
+        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ /^v?(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?(\+[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?/ }
```

Comment:
So in one case it's a branch name in other case it's a tag name, correct?

Reply:
Yeah. When the build is triggered from the tag Jenkins will set the env.BRANCH_NAME to the tag name.